tools/syntax_gen.py

```diff
@@ -118,7 +118,7 @@ def generate(outf, name, tags, members, alltypes):
 	outf.write('        {}({}{}){}\n'.format(base, kindValue, baseInitializers, initializers))
 	outf.write('    {\n')
 
-	if len(members) == 0:
+	if len(members) == 0 and not forceEmpty:
```

Comment:
PrimaryExpressionSyntax currently doesn't use final=false.  I assume I can set that to false?